src/pages/LoginPage.jsx

```diff
@@ -1,7 +1,148 @@
+                      id="email"
+                      className="appearance-none block w-full px-3 py-2 border border-gray-300 rounded-md shadow-sm placeholder-gray-400 focus:outline-none focus:ring-indigo-500 focus:border-indigo-500 sm:text-sm"
+                    />
+                  </div>
+                  <ErrorMessage name="email" component="div" className="mt-2 text-sm text-red-600" />
+                </div>
+                <div>
+                  <label htmlFor="password" className="block text-sm font-medium text-gray-700">
+                    Password*
+                  </label>
+                  <div className="mt-1">
+                    <Field
+                      type="password"
+                      name="password"
+                      id="password"
+                      className="appearance-none block w-full px-3 py-2 border border-gray-300 rounded-md shadow-sm placeholder-gray-400 focus:outline-none focus:ring-indigo-500 focus:border-indigo-500 sm:text-sm"
+                    />
+                  </div>
+                  <ErrorMessage name="password" component="div" className="mt-2 text-sm text-red-600" />
+                </div>
+                {!isLogin && (
+                  <div>
+                    <label htmlFor="confirmPassword" className="block text-sm font-medium text-gray-700">
+                      Confirm Password*
+                    </label>
+                    <div className="mt-1">
+                      <Field
+                        type="password"
+                        name="confirmPassword"
+                        id="confirmPassword"
+                        className="appearance-none block w-full px-3 py-2 border border-gray-300 rounded-md shadow-sm placeholder-gray-400 focus:outline-none focus:ring-indigo-500 focus:border-indigo-500 sm:text-sm"
+                      />
+                    </div>
+                    <ErrorMessage name="confirmPassword" component="div" className="mt-2 text-sm text-red-600" />
+                  </div>
+                )}
+                <div>
+                  <button
+                    type="submit"
+                    disabled={isSubmitting}
+                    className="w-full flex justify-center py-2 px-4 border border-transparent rounded-md shadow-sm text-sm font-medium text-white bg-indigo-600 hover:bg-indigo-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500"
+                  >
+                    {isLogin ? 'Sign in' : 'Register'}
+                  </button>
+                </div>
+              </Form>
+            )}
+          </Formik>
+          <div className="mt-6">
+            <button
+              onClick={() => setIsLogin(!isLogin)}
+              className="w-full flex justify-center py-2 px-4 border border-transparent rounded-md shadow-sm text-sm font-medium text-indigo-600 bg-white hover:bg-gray-50 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500"
```

Comment:
please change the background color from white to gay because the for color is white